pkgs/games/openra/engines.nix

```diff
@@ -1,4 +1,4 @@
-{ buildOpenRAEngine, fetchFromGitHub, extraPostFetch }:
+{ buildOpenRAEngine, fetchFromGitHub, postFetch }:
```

Comment:
Is this file ever invoked by users (i.e. outside of nixpkgs)? In that case they won't see a deprecation message this will just be a breaking change.

Reply:
as far as i can tell no it isn't